AppInsightsCommon/amd/types/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "applicationinsights-sdk-js-types",
+  "version": "0.0.1",
+  "description": "",
+  "license": "ISC"
```

Comment:
what is this?

Reply:
should the license refer to MIT license. I am not sure what is ISC

Reply:
default is ISC. I switched it to MIT